repo/blob/s3/s3_options.go

```diff
@@ -17,6 +17,7 @@ type Options struct {
 	Endpoint       string `json:"endpoint"`
 	DoNotUseTLS    bool   `json:"doNotUseTLS,omitempty"`
 	DoNotVerifyTLS bool   `json:"doNotVerifyTLS,omitempty"`
+	RootCA         string `json:"RootCA,omitempty"`
```

Comment:
Can we switch to byte slice which are automatically base64 encoded in json?

Reply:
Also json field should start with lowercase. Perhaps also include PEM in the name to indicate format.

Reply:
Case fixed, type changed, mention of PEM format has been moved to cli / env options that feeds this key (`root-ca-pem-path`, `root-ca-pem-base64`, `ROOT_CA_PEM_BASE64`)